recipes/onnxruntime/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "onnxruntime" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/microsoft/{{ name }}
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} tools/ci_build/build.py --build_dir build/Linux --config Release --use_openmp --use_mkldnn --enable_pybind --build_wheel && pip install build/Linux/Release/dist/*.whl"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git
+    - cmake
+  host:
+    - python
+    - numpy>=1.15
+  run:
+    - python
+    - numpy>=1.15
```

If you building against ``numpy`` it should be
```suggestion
    - {{ pin_compatible('numpy') }}
```